src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -33,7 +33,7 @@ export interface MenuItem {
 }
 
 export interface SubMenuItem {
-  id: string
+  id: string | number
```

Comment:
Thanks!  FYI for others: this is the one that's crucial for the multi-org work. Quartz returns arrays of `Accounts` with an id of type `number`, so we'd like to be able to us those as `subMenuItems` without having to re-map in the UI.